### DISCUSSION TITLE
Installing zigbee2mqtt as a service

### DISCUSSION THREAD

**Discussion by timdonovanuk** [tteck/Proxmox#585] [Help Needed] [answered]
Hi,

Most of your scripts install the app as a service, so that it's available if the LXC gets rebooted/restarted. 

For zigbee2mqtt (which literally _keeps the lights on_ ;)) you'd need to run `cd /opt/zigbee2mqtt && npm start` manually every time the LXC restarts, which is..not optimal. 

Thanks!

**Comment by tteck** [comment, marked as answer]
Nah, it creates a service `/etc/systemd/system/zigbee2mqtt.service`
https://github.com/tteck/Proxmox/blob/5f75108ff3cdab459688c756c49a0fea21a8c0ba/setup/zigbee2mqtt-install.sh#L97
Starts automatically at boot.

**Reply by guarddog13** [reply]
I was going to say I've reboot several times and haven't had this issue.
